frontend/src/components/Events/CreateEvents.js

```diff
@@ -71,9 +77,10 @@ function Create() {
         end: eventEnd,
         tags: tags,
         user: {
-          id: firebase.auth().currentUser.uid,
+          uid: firebase.auth().currentUser.uid,
```

Comment:
Can you use { currentUser } = useAuth() instead? And call it like so:
uid: currentUser.uid